SparkPost/2web2ui
# FE-1209 - [Dashboard] Add sending domain onboarding

Epic: https://sparkpost.atlassian.net/browse/FE-1185
Task: https://sparkpost.atlassian.net/browse/FE-1209

### What Changed
 - The current dashboardV2 onboarding panel is being broken up into individual steps the user will walk through depending on the state of their account/data. 

**Before:**
![image](https://user-images.githubusercontent.com/4204806/98590771-ec4e7100-2294-11eb-9dee-9366959e9789.png)

**After:** 
![image](https://user-images.githubusercontent.com/4204806/98588762-d4292280-2291-11eb-9544-7854789497b0.png)

Step one design:
https://sparkpost.invisionapp.com/console/Progressive-Onboarding-ckfe1rusj0cqn0185gnro5f8c/ckfe1s8be019p010e2zwj0wc6/play

Step four design:
https://sparkpost.invisionapp.com/console/Progressive-Onboarding-ckfe1rusj0cqn0185gnro5f8c/ckfe1s8bf019s010e68fy5lhe/play

### How To Test
 - Load up an admin or dev user (preferably both for good test coverage) and make sure it's a fresh account without a sending domain. Give the account the feature flag for `allow_domains_v2` and navigate to the dashboardV2 and you should see the first onboarding step for adding a sending domain. 

### To Do
- [x] Swap out the `<Picture />` component and use the correct image/media asset
- [x] Correct the conditions for the add sending domain boolean (onboarding step)
- [x] Add Cypress test
- [x] Split off work into individual PRs
